billing/tables/customer.js

```diff
@@ -56,8 +56,8 @@ export const createCustomerStore = (conf, { tableName }) => ({
         Key: marshall({ customer }),
         UpdateExpression: 'SET product = :product, updatedAt = :updatedAt',
         ExpressionAttributeValues: marshall({
-          product,
-          updatedAt: new Date().toISOString()
+          ':product': product,
+          ':updatedAt': new Date().toISOString()
```

Comment:
Ooo is that an existing bug?

Reply:
it is! the `PlanStorage` tests from `w3up` should have caught this but they weren't being run, since the test runner was not added before this PR